core/base/inc/ROOT/RConfig.hxx

```diff
@@ -571,6 +571,18 @@
 #endif
 #endif
 
+// See also https://nemequ.github.io/hedley/api-reference.html#HEDLEY_NEVER_INLINE
+// for other platforms.
+#ifdef R__HAS_ATTRIBUTE_NO_INLINE
+#define R__NEVER_INLINE inline __attribute__((noinline))
```

Comment:
Consider renaming to `R__NOINLINE` given that this is the spelling on all platforms that support this.

Reply:
I find NO_INLINE or NEVER_INLINE to be slight more readable ...

And there exists one platorm (http://ftp.iar.se/WWWfiles/arm/webic/doc/EWARM_DevelopmentGuide.ENU.pdf) where it is spelled ```#pragmainline=never``` :) [yep not one of ours :) ]

Reply:
and R__NEVER_INLINE is a 'good' opposite to R__ALWAYS_INLINE